src/Analytics/AnalyticsException.php

```diff
@@ -1,6 +0,0 @@
-<?php
```

Comment:
Why did we remove this? We now throw basic exceptions, or don't throw any?

Reply:
We don't throw any exceptions, we just log to console and return false